README.md

```diff
@@ -22,13 +22,17 @@ gradient-free optimizers, and constrained optimization.
 
 ### Installation
 
-ensmallen can be installed with CMake 3.3 or later.
-If CMake is not already available on your system, it can be obtained from https://cmake.org
+ensmallen can be installed in several ways.
 
+A straightforward approach is to simply copy the `include/ensmallen.hpp` header ***and*** the associated `include/ensmallen_bits` directory to a location such as `/usr/include/` which is searched by your C++ compiler.
+If you can't use `sudo` or don't have write access to `/usr/include/`, use a directory within your own home directory (eg. `/home/blah/include/`).
```

Comment:
It's nice to add this, but do you think you could move it below the CMake-based installation section?  I think in general we'd prefer users to use CMake, since it will check for dependencies like Armadillo before installing, and would reduce the likelihood of confused users who manually installed by copying the include files. :+1:

Reply:
Done. Automatic installation placed before manual installation in commit https://github.com/mlpack/ensmallen/commit/bb18d79413ef9cdee6f775d237500a0ff96c149c

Reply:
Clarified wording, as cmake based installation isn't really "automatic" (still need to type commands): https://github.com/mlpack/ensmallen/commit/0678b0518a4ceca2e23bc3231d2254f3df25e61d